packages/runtime/src/runtime/cloud/oauth.ts

```diff
@@ -50,9 +48,9 @@ export const createOauthTokenClient = (
   axios: AxiosInstance,
   oauthTokenClientProps: OauthTokenClientProps
 ) => async () => {
-  const { oauthUrl, clientId, clientSecret, scopes } = oauthTokenClientProps
+  const { clientId, clientSecret, scopes } = oauthTokenClientProps
   const res = await axios.post(
-    oauthUrl,
+    process.OAUTH_ENDPOINT!,
```

Comment:
Instead of adding the `!`, you should
```ts
if(!process.OAUTH_ENDPOINT) {
  throw new VError('OAUTH_ENDPOINT must be defined')
}
```
This will allow you to remove the `!` and provide a clear error message